htmd/queues/lsfqueue.py

```diff
@@ -259,30 +260,33 @@ def inprogress(self):
         if self.jobname is None:
             raise ValueError('The jobname needs to be defined.')
         user = getpass.getuser()
-        cmd = [self._qstatus, '-J', self.jobname, '-u', user, '-q', self.queue]
-        logger.debug(cmd)
-
-        # This command randomly fails so I need to allow it to repeat or it crashes adaptive
-        tries = 0
-        while tries < 3:
-            try:
-                ret = check_output(cmd)
-            except CalledProcessError:
-                if tries == 2:
-                    raise
-                tries += 1
-                time.sleep(3)
-                continue
-            break
-
-        logger.debug(ret.decode("ascii"))
-
-        # TODO: check lines and handle errors
-        l = ret.decode("ascii").split("\n")
-        l = len(l) - 2
-        if l < 0:
-            l = 0  # something odd happened
-        return l
+        l_total = 0
+        for q in ensurelist(self.queue):
+            cmd = [self._qstatus, '-J', self.jobname, '-u', user, '-q', q]
+            logger.debug(cmd)
+
+            # This command randomly fails so I need to allow it to repeat or it crashes adaptive
+            tries = 0
+            while tries < 3:
+                try:
+                    ret = check_output(cmd, stderr=DEVNULL)
+                except CalledProcessError:
+                    if tries == 2:
+                        raise
+                    tries += 1
+                    time.sleep(3)
+                    continue
+                break
+
+            logger.debug(ret.decode("ascii"))
+
+            # TODO: check lines and handle errors
+            l = ret.decode("ascii").split("\n")
```

Comment:
The variable name `l` is horrrrrible. But for this PR as it's urgent I will let it slide :P

Reply:
In my defense, it was already `l`